data/core/step1_get_started.md

```diff
@@ -476,18 +476,18 @@
     - utter_explain_core
     - utter_explain_nlu
 
-## not new to rasa + rasa
+## not new to rasa + rasax
 * how_to_get_started
     - utter_getstarted
     - utter_first_bot_with_rasa
 * deny
     - action_set_onboarding
     - slot{"onboarding": false}
     - utter_ask_which_product
-* faq{"product": "rasa"}
-    - slot{"product": "rasa"}
-    - utter_explain_core
-    - utter_explain_nlu
+* faq{"product": "x"}
+    - slot{"product": "x"}
+    - utter_explain_x
+    - utter_also_explain_nlucore
 
```

Comment:
I think this is the same as the new story you added below -- we should probably keep this one the same